util/feature/flag.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package feature
+
+import (
+	"strings"
+	"sync"
+
+	"github.com/Masterminds/semver"
+
+	"github.com/pingcap/tidb-dashboard/util/nocopy"
+)
+
+type Flag struct {
+	nocopy.NoCopy
+
+	Name string
+
+	once        sync.Once
```

Comment:
Is it possible to avoid having this `once`? You can maintain who is registered in the manager.